src/api/app/assets/stylesheets/webui/notifications_redesign/notifications.scss

```
@@ -1,19 +1,21 @@
 // TODO: As soon as notifications_redesign feature is 100% rolled out,
 // all the 'body.notifications-redesign' wrappers can be removed.
+$notifications-filter-box-height: 56px;
```

I suggest using a responsive unit:

```suggestion
$notifications-filter-box-height: 3.5rem;
```